src/SAWScript/Crucible/LLVM/Override.hs

```diff
@@ -507,32 +519,35 @@ methodSpecHandler opts sc cc top_loc css h = do
                       -- individually (concretely or symbolically) false
                       -- preconditions.
                       | not (null unsat) && null false && null symFalse ->
-                        [ PP.text (unwords
-                          [ "The conjunction of these overrides' preconditions"
-                          , "was unsatisfiable, meaning your override can never"
-                          , "apply. You probably have unintentionally specified"
-                          , "mutually exclusive/inconsistent preconditions."
-                          ]) PP.<$$>
-                          bullets '-' (unsat ^.. each . owpMethodSpec . to MS.ppMethodSpec)
+                        [ PP.vcat
+                          [ PP.pretty (unwords
```

Comment:
Might be useful to look at using prettyprinter's reflow here for this larger body of text.

Reply:
I can address this in #931, which builds on top of this PR.